automation/check-patch.req

```diff
@@ -0,0 +1,6 @@
+yum-utils
+autoconf
+automake
```

Comment:
Are `autoconf` and `automake` really needed? I don't see any `Makefile.am` or `configure.in` files, so I guess you don't really need those tools.